### PR TITLE
feat(ld-breadcrumbs): add breadcrumbs component

### LINKED ISSUE #128: Breadcrumbs
Screenshot 2021-11-22 at 10 59 52

See https://www.figma.com/file/JcDMeUwec9e185HfBgT9XE/Liquid-Oxygen?node-id=1084%3A14286

### PR DESCRIPTION
**Pull request by Tabanion**
# Description

Added new `ld-breadcrumbs` component.

Resolves #128

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] unit tests
- [x] e2e tests
- [x] accessibility tests
- [x] tested manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-link/ld-link.tsx [hunk 1]
```diff
@@ -79,6 +80,7 @@ export class LdLink implements ClonesAttributes {
         }
         rel={this.target === '_blank' ? 'noreferrer noopener' : undefined}
         disabled={this.disabled}
+        part="a"
```
Comment (renet) [inline src/liquid/components/ld-link/ld-link.tsx:83]: Can we please name it "link" or at least "anchor"? `::part(a)` may be a little confusing for a maintainer. - Also, shouldn't it also have the part "focusable" (and implement the InnerFocusable interface)?

**Review by renet — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-link/ld-link.css [hunk 2]
```diff
@@ -42,31 +51,54 @@
...
+        calc(0.5 * (var(--ld-link-gap) + var(--ld-link-chevron-gap)) + 50%)
+      )
+      translateY(-65%) rotate(-40deg);
+  }
+  &::after {
+    transform: translateX(
+        calc(-0.5 * var(--ld-link-gap) + 0.5 * var(--ld-link-chevron-gap) - 50%)
+      )
+      translateY(20%) rotate(40deg);
+  }
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/liquid/components/ld-link/ld-link.css:103]: The link PR has already been approve with this approach, which I still find is legit. I will not change this due to the reasons mentioned plus the additional effort to change everything.

[the remark this answers is not in the document]
Comment (renet) [reply src/liquid/components/ld-link/ld-link.css:103]: Please add the comment, as discussed in the review. Then this conversation can be resolved.

**Review by renet — APPROVED** [review]
LGTM subject to the open conversations.

**Pull request merged**